arm:
  kinematics_solver: kdl_kinematics_plugin/KDLKinematicsPlugin
  kinematics_solver_search_resolution: 0.005
  kinematics_solver_timeout: 0.005
  kinematics_solver_attempts: 3
whole_body:
  kinematics_solver: hsrb_moveit_kinematics/HSRBKinematicsPlugin
  kinematics_solver_search_resolution: 0.005
  kinematics_solver_timeout: 0.005
  kinematics_solver_attempts: 3
  # Joint weight for inverse kinematics
  #  arm_lift_joint, "arm_flex_joint, arm_roll_joint, wrist_flex_joint, wrist_roll_joint, odom_x, odom_y, odom_t
  joint_weight: [10.0, 1.0, 1.0, 1.0, 1.0, 10.0, 10.0, 100.0]
whole_body_weighted:
  kinematics_solver: hsrb_moveit_kinematics/HSRBKinematicsPlugin
  kinematics_solver_search_resolution: 0.005
  kinematics_solver_timeout: 0.005
  kinematics_solver_attempts: 3
  joint_weight: [10.0, 1.0, 1.0, 1.0, 1.0, 1000000000.0, 1000000000.0, 100.0]
whole_body_light:
  kinematics_solver: hsrb_moveit_kinematics/HSRBKinematicsPlugin
  kinematics_solver_search_resolution: 0.005
  kinematics_solver_timeout: 0.005
  kinematics_solver_attempts: 3
  joint_weight: [10.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 100.0]
